src/main.css

```diff
@@ -1,12 +1,14 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat');
```

Comment:
More of a query, do we want to use Google Fonts in production or?

Reply:
I can pull it down locally. That was the font that disappeared though 😄